diff --git a/llvm/unittests/TargetParser/RISCVISAInfoTest.cpp b/llvm/unittests/TargetParser/RISCVISAInfoTest.cpp
index b960e2ff51d8..0c9d833deb19 100644
--- a/llvm/unittests/TargetParser/RISCVISAInfoTest.cpp
+++ b/llvm/unittests/TargetParser/RISCVISAInfoTest.cpp
@@ -1002,31 +1002,31 @@ R"(All available -march extensions for RISC-V
     xcvmac               1.0
     xcvmem               1.0
     xcvsimd              1.0
-    xdma                0.1
-    xfalthalf           0.1
-    xfaltquarter        0.1
-    xfauxalthalf        0.1
-    xfauxaltquarter     0.1
-    xfauxhalf           0.1
-    xfauxquarter        0.1
-    xfauxvecalthalf     0.1
-    xfauxvecaltquarter  0.1
-    xfauxvechalf        0.1
-    xfauxvecquarter     0.1
-    xfauxvecsingle      0.1
-    xfexpauxvecalthalf  0.1
+    xdma                 0.1
+    xfalthalf            0.1
+    xfaltquarter         0.1
+    xfauxalthalf         0.1
+    xfauxaltquarter      0.1
+    xfauxhalf            0.1
+    xfauxquarter         0.1
+    xfauxvecalthalf      0.1
+    xfauxvecaltquarter   0.1
+    xfauxvechalf         0.1
+    xfauxvecquarter      0.1
+    xfauxvecsingle       0.1
+    xfexpauxvecalthalf   0.1
     xfexpauxvecaltquarter0.1
-    xfexpauxvechalf     0.1
-    xfexpauxvecquarter  0.1
-    xfquarter           0.1
-    xfrep               0.1
-    xfvecalthalf        0.1
-    xfvecaltquarter     0.1
-    xfvechalf           0.1
-    xfvecquarter        0.1
-    xfvecsingle         0.1
-    xmempool            0.1
-    xpulpv              2.0
+    xfexpauxvechalf      0.1
+    xfexpauxvecquarter   0.1
+    xfquarter            0.1
+    xfrep                0.1
+    xfvecalthalf         0.1
+    xfvecaltquarter      0.1
+    xfvechalf            0.1
+    xfvecquarter         0.1
+    xfvecsingle          0.1
+    xmempool             0.1
+    xpulpv               2.0
     xsfcease             1.0
     xsfvcp               1.0
     xsfvfnrclipxfqf      1.0
@@ -1035,7 +1035,7 @@ R"(All available -march extensions for RISC-V
     xsfvqmaccqoq         1.0
     xsifivecdiscarddlone 1.0
     xsifivecflushdlone   1.0
-    xssr                0.1
+    xssr                 0.1
     xtheadba             1.0
     xtheadbb             1.0
     xtheadbs             1.0
